packages/timezone-data/lib/index.js

```diff
@@ -132,7 +132,7 @@ export default [
         label: '(GMT +2:00) Harare'
     },
     {
-        name: 'Europe/Kyiv',
+        name: 'Europe/Kiev',
```

Comment:
Cool. Did we make sure this part doesn't show up anywhere in the UI? As in we only use `name` and not the `label` to display things?

Reply:
Also, let's add a comment of why we have put in the wrong name back and how can we fix this in the future 🙏

Reply:
Yes, the label is used in the UI, I have checked the code for this.